typescript/src/ile/sum.sql

```diff
@@ -0,0 +1,5 @@
+create or replace procedure X.sumpgm (IN numa INT, IN numb INT, OUT result INT) 
+LANGUAGE RPGLE  
+EXTERNAL NAME X.SUM GENERAL;
+
+call X.SUMPGM(1, 2, default);
```

Comment:
Updated `X` to `TESTING` and ran this using the `RUNSQLSTM` action, but I am getting this:
```
SQL7989   0       1  Position 1 Procedure SUMPGM was created in TESTING.
SQL0469  30       5  Position 1 IN, OUT, or INOUT not valid for parameter 3
                      in procedure SUMPGM in TESTING.
```

Am I doing something obviously wrong?

Reply:
Pass in `0` instead of `default`. It might work then.

Reply:
Tried that as well. Gives the same error

Reply:
Yeah same here. Must only be callable with a binding parameter (see `routes/root.ts`)

Reply:
@SanjulaGanepola This worked:

```sql
create or replace variable liama.outparm int default 0;
call LIAMA.SUMPGM(1, 2, liama.outparm);
```

Reply:
Yup that fixed it, thanks